src/segment-loader.js

```diff
@@ -635,6 +635,31 @@ export default class SegmentLoader extends videojs.EventTarget {
       this.roundTrip = request.roundTripTime;
       this.bandwidth = request.bandwidth;
       this.bytesReceived += request.bytesReceived || 0;
+      this.trigger({
+        type: 'stat',
+        data: {
+          name: 'numberOfBytesTransferred',
+          amount: request.bytesReceived || 0
+        }
+      });
+
+      // if we get this far there are no errors,
+      // increment the stat for media requests
+      this.trigger({
+        type: 'stat',
+        data: {
+          name: 'numberOfMediaRequests',
```

Comment:
We could also look at how [MSE exposed video quality](https://w3c.github.io/media-source/#htmlvideoelement-extensions) for inspiration on how to expose this info in a way that fits with the rest of the platform.